resolwe_api/resolwe.py

```diff
@@ -14,7 +14,9 @@
     from urllib import parse as urlparse
 
 import requests
+from requests.exceptions import ConnectionError
```

Comment:
Use one or the other option. If you really wish to use it bot ways, group import and from sentences separately. Read about structuring imports [here](https://www.python.org/dev/peps/pep-0008/#imports).